scripts/send-json-api-requests.sh

```diff
@@ -37,10 +37,5 @@ do
     getParty
   fi
 
-  # Generate traffic on the health check endpoints
```

Comment:
It makes sense to generate traffic on the health endpoints as well, but not necessarily in every iteration.